contracts/mocks/MockSablierV2LockupLinear.sol

```diff
@@ -6,20 +6,7 @@ import "../interfaces/sablier/ISablierV2LockupLinear.sol";
 import {LockupLinear} from "../interfaces/sablier/LockupLinear.sol";
 
 contract MockSablierV2LockupLinear is ISablierV2LockupLinear {
-    // Define the Stream struct here
-    struct Stream {
-        address sender;
-        address recipient;
-        uint128 totalAmount;
-        address asset;
-        bool cancelable;
-        bool transferable;
-        uint40 startTime;
-        uint40 cliffTime;
-        uint40 endTime;
-    }
-
-    mapping(uint256 => Stream) public streams;
```

Comment:
Ah yeah, maybe it does make sense to keep this struct defined here... (considering my first comment)